spec/cddl/many.cddl

```diff
@@ -169,45 +158,4 @@ attribute-id-standard = uint
 attribute-id-custom = nint
 
 ; An attribute contains an ID and arguments.
-attribute = attribute-id / [attribute-id, * attribute-arg]
-
-; Represents a parent operation in a proof
-parent = h'10'
-
-; Represents a child operation in a proof
-child = h'11'
-
-; The hash of a key-value pair
-key-value-hash = bstr
-
-; The hash of a node in a proof, containing, in order, the hash
-; of the key-value pair, the hash of the left child, and the hash of
-; the right child
-node-hash = bstr
-
-; A node key of a proof is a collection of bytes together with its length
-node-key = {
-    ; The length of the key
-    0 => uint,
-
-    ; The value of the key
-    1 => bstr,
-}
-
-; A node value of a proof is a collection of bytes together with its length
-node-value = {
-    ; The length of the value
-    0 => uint,
-
-    ; The value itself
-    1 => bstr
-}
-
-; The key-value pair of a proof
-key-value-pair = {
-    ; The key
-    0 => node-key,
-
-    ; The value
-    1 => node-value
-}
\ No newline at end of file
+attribute = attribute-id / [attribute-id, * attribute-arg]
```

Comment:
EOL EOF

Reply:
I'm going to reiterate this. It's nicer for diffs to have a newline at the end of a file.

I'm also okay if you want to add an `.editorconfig` or `.env` file to this repo.